app/models/premium_table.rb

```diff
@@ -7,6 +7,7 @@ class PremiumTable
   field :start_on, type: Date
   field :end_on, type: Date
   field :cost, type: Float
+  field :rating_area, type: String
```

Comment:
They will provide an XML file, and we already have a rake test for that, once we run that rake test with given XML file, Plans and Premium_table data will get loaded.